components/TokenBalance/TokenDeposit.tsx

```diff
@@ -101,29 +103,45 @@ export const TokenDeposit = ({
     return null
   }
 
+  const voterWeightAddin  =
+    tokenRole === GoverningTokenRole.Community
+      ? config?.account.communityTokenConfig.voterWeightAddin
+      : config?.account.councilTokenConfig.voterWeightAddin
+
+  const plugin = findPluginName(voterWeightAddin);
+
   return (
     <div className="w-full">
       {(availableTokens != '0' || inAccountDetails) && (
         <div className="flex items-center space-x-4">
           {tokenRole === GoverningTokenRole.Community ? (
-            <VanillaVotingPower className="w-full" role="community" />
+            plugin === 'pyth' ? (
+              <PythVotingPower className="w-full" role='community' />
+            ) :
+              <VanillaVotingPower className="w-full" role="community" />
           ) : (
             <VanillaVotingPower className="w-full" role="council" />
           )}
         </div>
       )}
 
       <div
-        className={`my-4 opacity-70 text-xs  ${
-          canShowAvailableTokensMessage ? 'block' : 'hidden'
-        }`}
+        className={`my-4 opacity-70 text-xs  ${canShowAvailableTokensMessage ? 'block' : 'hidden'
+          }`}
       >
         You have {tokensToShow} {hasTokensDeposited ? `more ` : ``}
-        {depositTokenName} tokens available to deposit.
+        {depositTokenName} tokens available to deposit. {plugin === 'pyth' ? 'You can deposit them at https://staking.pyth.network/ . ' : ""}
+      </div>
+
+      <div
+        className={`my-4 opacity-70 text-xs  ${(plugin === 'pyth') ? 'block' : 'hidden'
+          }`}
+      >
+        If you previously deposited tokens on this page, use the button below to withdraw them immediately. Those tokens have no voting power.
       </div>
 
       <div className="flex flex-col mt-6 space-y-4 sm:flex-row sm:space-x-4 sm:space-y-0">
-        {hasTokensInWallet || inAccountDetails ? (
+        {(hasTokensInWallet || inAccountDetails) && plugin !== "pyth" ? (
```

Comment:
Disable Deposit token for Pyth, since they need to deposit on our staking website.